pages/linux/tracepath.md

```diff
@@ -0,0 +1,28 @@
+# tracepath
+
+> Trace the path to a network host discovering MTU along this path.
+> More information: <https://manned.org/tracepath>.
+
+- A preferred way to trace the path to a host:
+
+`tracepath -p 33434 {{host}}`
+
+- Specify the initial destination port, useful with non-standard firewall settings:
+
+`tracepath -p {{destination_port}} {{host}} `
+
+- Print numerical IP addresses only:
+
+`tracepath -n {{host}}`
```

Comment:
On a second thought, I think this is not that much useful option, compared to the `-b`. 

It could potentially be removed or replaced with `-6`, see https://github.com/tldr-pages/tldr/pull/5744/files#r613359397